probePlugin/src/main/scala/org/virtuslab/ideprobe/handlers/BackgroundTasks.scala

```diff
@@ -1,11 +1,12 @@
 package org.virtuslab.ideprobe.handlers
 
 import java.lang.reflect.Method
-import com.intellij.openapi.progress.ProgressIndicator
-import com.intellij.openapi.progress.ProgressManager
+import com.intellij.openapi.progress.{ProgressIndicator, ProgressManager}
+import org.virtuslab.ideprobe.Extensions._
+import org.virtuslab.ideprobe.protocol.AwaitIdle
 import scala.annotation.tailrec
 import scala.concurrent.duration._
-import org.virtuslab.ideprobe.Extensions._
+import pureconfig.generic.auto._
```

Comment:
make sure that the class that uses this import extends from ConfigFormat (I don't remember if IntelliJApi does)

Reply:
that must be PayloadJsonFormat